### DISCUSSION TITLE
Scalebar in screenshot

### DISCUSSION THREAD

**Discussion by aburfeid** [biigle/core#571] [Ideas]
Hi Everyone,

we were thinking, that it would be great if the BIIGLE-screenshot was able to contain the scalebar. We can of course use the snipping tool or screenshot functions of the computer, but the other BIIGLE-tools are still visible.

I have searched for a similar topic but found none.

Thanks for your help and thanks for BIIGLE!

Best wishes,

Andrea

**Comment by mzur** [comment]
This was moved to a feature idea in https://github.com/biigle/core/issues/573.
